fontbe/src/os2.rs

```diff
@@ -813,6 +816,25 @@ impl Work<Context, AnyWorkId, Error> for Os2Work {
     fn exec(&self, context: &Context) -> Result<(), Error> {
         let static_metadata = context.ir.static_metadata.get();
 
+        // We set the OS/2.us{Weight,Width}Class to the default value of 'wght'/'wdth' axes
+        // in the same way fontmake does when building a VF:
+        // https://github.com/fonttools/fonttools/blob/770917d8/Lib/fontTools/varLib/__init__.py#L1016-L1032
+        let default_wght = static_metadata
+            .axes
+            .iter()
+            .find(|axis| axis.tag == Tag::new(b"wght"))
+            .map(|axis| axis.default.into_inner().0)
+            .unwrap_or(400.0);
+        let us_weight_class: u16 = default_wght.clamp(1.0, 1000.0).ot_round();
+
+        let default_wdth = static_metadata
+            .axes
+            .iter()
+            .find(|axis| axis.tag == Tag::new(b"wdth"))
+            .map(|axis| axis.default.into_inner().0)
+            .unwrap_or(100.0);
```

Comment:
These lines repeat almost identically here and above. I suggest extracting a function or even adding it to static metadata, e.g. `static_metadata.axis(Tag) -> Option<Axis>`